helm-frontend/src/components/Landing/GlobalLanding.tsx

```diff
@@ -70,12 +68,19 @@ export default function LegacyLanding() {
   return (
     <>
       <SimpleHero />
-      <div className="container mt-40 mx-auto text-lg px-16">
+      <div className="container mt-40 mx-auto text-lg">
         <div className="flex flex-col sm:flex-row justify-center mb-10 flex sm:gap-8 md:gap-32">
           <h1 className="text-4xl  mx-4 ">
             <strong>HELM Projects</strong>
           </h1>
         </div>
+        <div className="flex flex-col sm:flex-row flex sm:gap-8 md:gap-32">
+          <text>
+            HELM projects involve utilizing the HELM framework on specific
```

Comment:
How about: HELM projects leverage the HELM framework and target particular domains, languages, or use cases.